Extension/package.json

```diff
@@ -455,6 +455,12 @@
           "default": null,
           "description": "The value to use for the system include path. If set, it overrides the system include path acquired via \"compilerPath\" and \"compileCommands\" settings.",
           "scope": "resource"
+        },
+        "C_Cpp.disableLanguageService": {
```

Comment:
I was planning to expose this differently, via `C_Cpp.intelliSenseEngine` (until we refactor settings and have `intelliSense.enabled`).

Reply:
You mean a "Disabled" setting for intelliSenseEngine? I could do that.